### DISCUSSION TITLE
Cursor line extends across Panes

### DISCUSSION THREAD

**Discussion by vbfischer** [LazyVim/LazyVim#183] [Q&A]
I did something, not sure what yet... But the cursor line extends across the entire width of the terminal, across each pane (see screenshot below). I'm searching for how to revert it, but haven't come across how I did it or how to fix it...

![CleanShot 2023-02-05 at 20 39 15](https://user-images.githubusercontent.com/138106/216862563-6ece995c-74da-4f26-b640-aacdfffbf9c3.png)

**Comment by folke** [comment]
No idea what that is, but it defintely isn't coming from LazyVim

**Reply by vbfischer** [reply]
Yea pretty sure it's something I did. Haven't customized anything other than a few key commands. Likely just hit a random key command that did something and was hoping someone recognized it

**Reply by folke** [reply]
Probably something in tmux. That doens't look like a Neovim thing

**Reply by vbfischer** [reply]
Oooh that's a variable I forgot about. Thanks for noticing, I'll check that out

**Reply by vbfischer** [reply]
ok... I think it was an iTerm2 thing... as I *THOUGHT* i had tried restarting iTerm before, but I noticed when I quit out of both nvim and tmux it was still showing the extended cursor line... But, restarting iTerm fixed it.. Wish I knew what errant key command I used to get in that state to begin with...
